Tests/test_imagefontctl.py

```diff
@@ -104,6 +104,22 @@ def test_text_direction_rtl2(self):
 
         self.assert_image_similar(im, target_img, .5)
 
+    def test_text_direction_ttb(self):
+        ttf = ImageFont.truetype(FONT_PATH, FONT_SIZE)
+
+        im = Image.new(mode='RGB', size=(100, 300))
+        draw = ImageDraw.Draw(im)
+        try:
+            draw.text((0, 0), 'English عربي', font=ttf, fill=500, direction='ttb')
```

Comment:
Wouldn’t it be better to actually test some CJK character here?

Reply:
I'm having trouble finding a small CJK font that could be included within the Pillow license. Do you have any thoughts?

Reply:
Actually, I've found one that's not too large.

Reply:
You can also subset any font with `pyftsubset` (from [FontTools](https://github.com/fonttools/fonttools)) to include just the few characters used in the test.